x/txfees/keeper/feetokens_test.go

```diff
@@ -65,7 +121,7 @@ func (s *KeeperTestSuite) TestUpgradeFeeTokenProposals() {
 		{
 			name:       "proposal with non-existent pool",
 			feeToken:   "foo",
-			poolId:     100000000000,
+			poolId:     10000000,
```

Comment:
Why was this changed?

Reply:
in pool manager keeper we only use 12 bytes for storing pool id, this requires 13 bytes & causes panic in tests thus needs to be lowered!